subgraphs/abyssmal/abis/Abyssmal.json

```diff
@@ -521,7 +516,7 @@
         "type": "string"
       }
     ],
-    "stateMutability": "view",
+    "stateMutability": "pure",
```

Comment:
Curious why you adjusted this from `view` to `pure`

Reply:
I think I was using an old ABI here, I didn't change it